plugins/source/shopify/resources/services/checkout/abandoned_checkouts_fetch.go

```diff
@@ -10,18 +13,52 @@ import (
 func fetchAbandonedCheckouts(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
 	cl := meta.(*client.Client)
 
+	const key = "abandoned_checkouts"
+
+	p := url.Values{}
+	min := time.Time{}
+
+	if cl.Backend != nil {
+		value, err := cl.Backend.Get(ctx, key, cl.ID())
+		if err != nil {
+			return fmt.Errorf("failed to retrieve state from backend: %w", err)
+		}
+		if value != "" {
+			min, err = time.Parse(time.RFC3339, value)
+			if err != nil {
+				return fmt.Errorf("retrieved invalid state value: %q %w", value, err)
+			}
+			p.Set("updated_at_min", min.Format(time.RFC3339))
```

Comment:
It looks like this key gets set but never read? Or maybe I'm just overlooking it on the GitHub interface, not sure

Reply:
It's read by the shopify api endpoint https://shopify.dev/api/admin-rest/2022-10/resources/abandoned-checkouts#get-checkouts?limit=1